README.md

```diff
@@ -131,17 +135,52 @@ Or you can run the native executable build for Linux in a container using:
 mvn package -Pnative -Dquarkus.native.container-build=true
 ```
 
-You can then execute your native executable with: `./target/strimzi-cistic-odpadu-1.0.0-SNAPSHOT-runner`.
+This is useful especially when running on other operating systems such as macOS or when you don't have GraalVM installed.
 
-### Building a container image
+You can then execute your native executable with: `./target/strimzi-drain-cleaner-1.0.0-SNAPSHOT-runner`.
 
-You can build the container for example using the _distro-less_ base image (use your own repository ;-)):
+### Creating a native executable using Make
+
+If you have [GraalVM](https://www.graalvm.org/) installed locally, you can create a native executable using: 
+```shell script
+make java_package
+```
+
+Or you can run the native executable build for Linux in a container using: 
+```shell script
+MVN_ARGS=-Dquarkus.native.container-build=true make java_package
+```
+
+This is useful especially when running on other operating systems such as macOS or when you don't have GraalVM installed.
+
+You can then execute your native executable with: `./target/strimzi-drain-cleaner-1.0.0-SNAPSHOT-runner`.
+
+### Building a container image manually
+
+After you have the native executable, you can build the container manually:
 
 ```
-docker build -f src/main/docker/Dockerfile.native-distroless -t quay.io/scholzj/strimzi-cistic-odpadu:latest .
-docker push quay.io/scholzj/strimzi-cistic-odpadu:latest
+docker build -f Dockerfile -t my-registry.tld/my-org/strimzi-drain-cleaner:latest .
+docker push my-registry.tld/my-org/strimzi-drain-cleaner:latest
 ```
 
+_Update the container image name to match your own registry / organization etc._
+
+### Building a container image using Make
+
+You can also build the image and push it into the registry using Make:
+
+```
+make docker_build docker_push
+```
+
+You can use the following environment variables to configure where will the image be pushed:
+* `DOCKER_REGISTRY` defines the registry where it will be pushed. 
+  For example `docker.io`.
```

Comment:
wouldn't be better to have there quay as we use it across whole org?

Reply:
I think `docker.io` is better example because it is known by more people. It gives them better idea what this env var stands for.